include/vcpkg/base/messages.h

```diff
@@ -751,6 +783,12 @@ namespace vcpkg
     DECLARE_MESSAGE(ExpectedPortName, (), "", "expected a port name here");
     DECLARE_MESSAGE(ExpectedTripletName, (), "", "expected a triplet name here");
     DECLARE_MESSAGE(ExtendedDocumentationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
+    DECLARE_MESSAGE(FailedToParseBinParagraph,
+                    (msg::url, msg::error_msg, msg::value),
+                    "'{error_msg}' is the error message for failing to parse the Binary Paragraph, '{value}' is the "
+                    "Binary Paragraph",
+                    "[sanity check] Failed to parse a serialized binary paragraph.\nPlease open an issue at {url}, "
```

Comment:
Is there a different URL we would ever use here or should it just be part of the message?

Reply:
What if the url changes in the future?